index.html

```diff
@@ -102,24 +102,24 @@ <h1 id = "page-title" class = "drop-shadow-2xl text text-center font-Rowdies tex
             <br>
             <button id="history-button" class=" dark:md:hover:bg-yellow-50 p-1.5 bg-orange-300 text-black font-thin">View Historical Results</button>
             <br>
-            <button id="clear-history">clear history</button> <!--used for testing-->
+            <button id="clear-history">clear history</button> 
         </div>
         </section>
     <br>
     
-        <!-- Current Results -->
-        <section>
-            <h2>SYNOPSIS</h2>
-            <div id="movieDisplay"></div>
-            <p id="messageDisplay"></p>
-            <br>
-            <h2>VIDEO PREVIEW</h2>
-            <!-- Note: src attribute URL is placeholder. Needs to be removed -->
-            <iframe id="youtube-preview"
-                width="640" height="360"
-                frameborder="0"
-            ></iframe>
-        </section>
+    <div>    
+        <div class="after-result">
+        <button id="return-button">Back to Search!</button>
+        <br>
+        <button id="retry-button">Try Again!</button>
+    </div>
+
+
+
+
+        <div id="movieDisplay"></div>
+
+        <p id="messageDisplay"></p>
```

Comment:
I think if we replace lines 120-122 here with the red lines 110-122 it should fix the problem